File: Python/0088_merge_sorted_array/solution.py
def merge_sorted_array(nums1,m,nums2,n):
    result = nums1[:m] + nums2[:n]
    result.sort()
    for index in range(len(result)):
        nums1[index] = result[index]
    return nums1

nums1, m = [1,2,3,0,0,0], 3
nums2, n = [2,5,6], 3
print(merge_sorted_array(nums1,m,nums2,n))

File: JavaScript/0088_merge_sorted_array/solution.js
function  merge_sorted_array(nums1,m,nums2,n) {
    let result = [...nums1.slice(0,m), ...nums2.slice(0,n)];
    result = result.sort((a,b) => a - b);
    for (let index = 0; index < result.length; index ++) {
        nums1[index] = result[index];
    }
    return nums1;
}

const nums1 = [1,2,3,0,0,0];
const m = 3;
const nums2 = [2,5,6];
const n = 3;
console.log(merge_sorted_array(nums1,m,nums2,n));